### PR TITLE
Amend API method template to describe `DOMException` usage

### PR DESCRIPTION
**Pull request by teoli2003**
This is our current practice, but I never documented it.

### CODE REVIEW

File: files/en-us/mdn/structures/page_types/api_method_subpage_template/index.md [hunk 1]
```diff
@@ -111,7 +111,14 @@ If the method doesn't return anything, just put "{{jsxref('undefined')}}.".
 - Exception2
   - : etc.
 
-## Examples
+Note that we have two kinds of exceptions: {{domxref("DOMException")}} objects and regular JavaScript exceptions, like {{jsxref("TypeError")}} and {{jsxref("RangeError")}}. A web developer needs to know which object is thrown, and for exceptions that are `DOMException` objects, he also needs to know their `name`. Here is an example where a method can raise a `DOMException` with a name of `IndexSizeError`, a `DOMException` with a name of `SyntaxError` and a JavaScript exception of type `TypeError`:
```
Comment (wbamberg) [inline files/en-us/mdn/structures/page_types/api_method_subpage_template/index.md:114]:
```suggestion
Note that we have two kinds of exceptions: {{domxref("DOMException")}} objects and regular JavaScript exceptions, like {{jsxref("TypeError")}} and {{jsxref("RangeError")}}. A web developer needs to know:

 - which object is thrown
 - for exceptions that are `DOMException` objects, the `name` of the exception
 
Here is an example where a method can raise a `DOMException` with a name of `IndexSizeError`, a `DOMException` with a name of `SyntaxError` and a JavaScript exception of type `TypeError`:
```

**Review by wbamberg — CHANGES_REQUESTED** [review]
One copy edit for clarity and gender-neutrality.

File: files/en-us/mdn/structures/page_types/api_method_subpage_template/index.md (hunk 1 above)
Comment (teoli2003) [reply files/en-us/mdn/structures/page_types/api_method_subpage_template/index.md:114]: I try to avoid he, but I missed this one.

**Review by wbamberg — APPROVED** [review]
👍 thank you @teoli2003 !

**Pull request merged**